src/components/password/password-test.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import Password, { PasswordProps } from ".";
+
+export default {
+  title: "Password/Test",
+  includeStories: ["Default"],
+  parameters: {
+    info: { disable: true },
+    chromatic: {
+      disableSnapshot: true,
+    },
+  },
+  argTypes: {
```

Comment:
**suggestion:** I agree we shouldn't surface every potential prop as an argType, as consumers will probably only use a few, and I think your used props are pretty much spot on. The only other ones I'd suggest adding are `inputHint` and maybe `prefix` at a push. Imo `inputHint` should definitely be added as it's a key part of the `Password component`